asreview/state/utils.py

```
@@ -122,11 +127,43 @@ def state_from_file(data_fp):
         logging.error(f"File {data_fp} does not exist, cannot create state.")
         return None
 
-    if not Path(data_fp).suffix in STATE_EXTENSIONS:
-        logging.error(f"file {data_fp} does not end with {STATE_EXTENSIONS}.")
-        return None
+    if Path(data_fp).suffix == ".asreview":
+        base_state = state_from_dot_asreview(data_fp)
+    else:
+        if not Path(data_fp).suffix in STATE_EXTENSIONS:
+            logging.error(f"file {data_fp} does not end with {STATE_EXTENSIONS}.")
+            return None
+        base_state = _get_state_class(data_fp)(state_fp=data_fp, read_only=True)
     state = {
         os.path.basename(os.path.normpath(data_fp)):
-        _get_state_class(data_fp)(state_fp=data_fp, read_only=True)
+        base_state
     }
     return state
+
+
+def state_from_dot_asreview(data_fp):
+    """Obtain the state from a .asreview file.
+
+    Parameters
+    ----------
+    data_fp: str
+        Path to .asreview file.
+
+    Returns
+    -------
+    BaseState:
+        The same type of state file as in the .asreview file, which at the moment
+        is JSONState.
+    """
+    if not Path(data_fp).suffix == '.asreview':
+        logging.error(f"file {data_fp} does not end with '.asreview'.")
```

Raise ValueError instead of logging.